File: output/story-learning/foundation-20260906/run/a-direct-v2-state/.tmp/plugins/plugins/plugin-eval/fixtures/ts-python-sample/src/sample.py
def choose_path(items, include_archived, prefer_fast):
    next_items = []
    for item in items:
        if not item:
            continue
        if include_archived or not item.startswith("archived:"):
            if prefer_fast and "fast" in item:
                next_items.append(item.upper())
            elif (not prefer_fast) and "slow" in item:
                next_items.append(item.lower())
            elif len(item) > 3:
                next_items.append(item)
    return next_items

File: research/2026-09-08-charlie-course-cleaning/native-supplements/worker-syntax-high-4-state/.tmp/plugins/plugins/plugin-eval/fixtures/ts-python-sample/src/sample.ts
export function choosePath(items: string[], includeArchived: boolean, preferFast: boolean): string[] {
  const next: string[] = [];
  for (const item of items) {
    if (!item) {
      continue;
    }
    if (includeArchived || !item.startsWith("archived:")) {
      if (preferFast && item.includes("fast")) {
        next.push(item.toUpperCase());
      } else if (!preferFast && item.includes("slow")) {
        next.push(item.toLowerCase());
      } else if (item.length > 3) {
        next.push(item);
      }
    }
  }
  return next;
}
